kolibri/core/content/utils/search.py

```diff
@@ -4,10 +4,17 @@
 """
 import hashlib
 
+try:
+    from django.contrib.postgres.aggregates import BitOr
+except ImportError:
+    BitOr = None
```

Comment:
Would this ever actually fail to import?

Reply:
yes
`ImportError: No module named psycopg2.extras`
in python 2.7